src/ontology/mondo-edit.obo

```diff
@@ -505014,8 +505003,6 @@ xref: OMIM:620040 {source="MONDO:equivalentTo"}
 xref: UMLS:C5774217 {source="MEDGEN:1823990", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0015780 {source="OMIM:620040"} ! dyskeratosis congenita
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12441 {source="OMIM:620040"} ! TYMS
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/30365 {source="OMIM:620040"} ! ENOSF1
```

Comment:
This term has the annotation
`excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql`

These gene annotations should not be removed.